draft-ietf-webpush-protocol.xml

```diff
@@ -453,9 +453,15 @@ Location: https://push.example.net/d/qDIYHNcfAIPP_5ITvURr-d6BGtYnTRnk
 ]]></artwork>
       </figure>
       <t>
-        A push service MAY generate a 413 (Payload Too Large) status code in
-        response to requests that include an entity body that is too large.
-        Push services MUST NOT generate a 413 status code in responses to an
+        A push service MAY return a 429 (Too Many Requests) status code <xref target="RFC6585"/>
+        when an application server has exceeded its rate limit for push message
+        delivery. The push service SHOULD also include a Retry-After header <xref target="RFC7231"/>
+        to indicate how long the application server must wait before it makes another request. 

```

Comment:
Thanks for the guidance on "must".

If scope is determined by the push service, then how would a application server understand how to interpret whether the policy was targeted at the application server in general or a specific URL managed by the application server? Are you suggesting some type of out-of-band policy/contract documented by the PS to the AS?